package.json

```diff
@@ -1,105 +1,43 @@
 {
-  "name": "waku",
-  "description": "Minimalistic React Framework",
-  "version": "0.14.0",
+  "name": "waku-monorepo",
+  "version": "0.1.0",
   "type": "module",
-  "author": "Daishi Kato",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dai-shi/waku.git"
-  },
-  "exports": {
-    "./package.json": "./package.json",
-    ".": {
-      "types": "./dist/main.d.ts",
-      "require": "./dist/cjs/main.js",
-      "default": "./dist/main.js"
-    },
-    "./node-loader": {
-      "types": "./dist/node-loader.d.ts",
-      "require": "./dist/cjs/node-loader.js",
-      "default": "./dist/node-loader.js"
-    },
-    "./config": {
-      "types": "./dist/config.d.ts",
-      "require": "./dist/cjs/config.js",
-      "default": "./dist/config.js"
-    },
-    "./client": {
-      "types": "./dist/client.d.ts",
-      "require": "./dist/cjs/client.js",
-      "default": "./dist/client.js"
-    },
-    "./server": {
-      "types": "./dist/server.d.ts",
-      "require": "./dist/cjs/server.js",
-      "default": "./dist/server.js"
-    },
-    "./router/client": {
-      "types": "./dist/router/client.d.ts",
-      "require": "./dist/cjs/router/client.js",
-      "default": "./dist/router/client.js"
-    },
-    "./router/server": {
-      "types": "./dist/router/server.d.ts",
-      "require": "./dist/cjs/router/server.js",
-      "default": "./dist/router/server.js"
-    }
-  },
-  "bin": {
-    "waku": "./dist/cli.js"
-  },
-  "files": [
-    "src",
-    "dist"
-  ],
+  "private": true,
   "scripts": {
-    "dev": "swc src -d dist -w",
-    "compile": "rm -rf dist && npm run compile:code && npm run compile:types",
-    "compile:code": "swc src -d dist && swc src -d dist/cjs -C module.type=commonjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json",
-    "compile:types": "tsc --project tsconfig.build.json",
+    "dev": "pnpm -r --filter='./packages/*' run dev",
+    "compile": "pnpm -r --filter='./packages/*' run compile",
+    "postinstall": "test -d ./packages/waku/dist || (pnpm -r --filter='./packages/waku' run compile && pnpm link ./packages/waku)",
```

Comment:
Wow you don't need pnpm link. https://pnpm.io/workspaces#workspace-protocol-workspace

Reply:
I would like to avoid `pnpm link` if possible.
Our goal here is to keep package.json in `examples/*` **unchanged** and allow this:
```bash
git clone ...
pnpm install 
pnpm run examples:dev:01_counter
```